docker/pytorch-aarch64/patches/torch-xla.patch

```diff
@@ -0,0 +1,28 @@
+From b2670989f2b74f0f33c83ba18912f8a131814a77 Mon Sep 17 00:00:00 2001
```

Comment:
This file will need the appropriate Copyright notice - as per the other .patch files in this and the TF build.